### ISSUE TITLE
Different value on dropdown

### ISSUE THREAD

**Issue by Drex-luthor** [patrickkunka/easydropdown#89]
Is it possible to have a different text displayed when the drop-down is selected?

Have it say "Afghanistan" on the drop-down but "AF" when selected.

**Comment by patrickkunka** [comment]
Hi there - I'm afraid this isn't possible currently, as easydropdown follows the behaviour of the native `<select>` element here.

I'd be open to adding a new API for this if you think it would be useful?

**Comment by Drex-luthor** [comment]
Yes, I see. I think it would be very helpful to have something of this sort to have more control of the select element. If possible it would be great to add this as a feature.

**Comment by tevdevelops** [comment]
I'm actually trying to implement this same exact feature and I think it would be very helpful and continue to add to the customization of this library. I am trying to hack around by using the api events but it seems your methods to update the dropdown text are overwriting my work

**Comment by 7iomka** [comment]
Any news?

**Comment by yoktav** [comment]
I guess @patrickkunka is busy
